src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${robot.coords.x}, y=${robot.coords.y}`;
+    },
+
+    goForward(y = 1) {
+      if (y > 0) {
+        robot.coords.y += y;
+      }
+
+      return robot;
+    },
```

Comment:
```suggestion
    goForward(y = 1) {
      if (y > 0) {
        this.coords.y += y;
      }

      return this;
    },
```
You can use this keyword